src/api-review-state.ts

```diff
@@ -124,10 +132,35 @@ export async function addOrUpdateAPIReviewCheck(
 
   debug(`Found ${comments.length} API reviews from WG members`);
 
+  // Combine reviews/comments and filter by recency.
   const allReviews = [
-    ...new Map([...comments, ...reviews].map((item) => [item.user.id, item])).values(),
+    ...[...comments, ...reviews]
+      .reduce((hash, item) => {
+        if (!/API LGTM|API DECLINED/.test(item.body)) return hash;
```

Comment:
the other regex tests `/gi`, so this should too. Even better would be to not duplicate the regex :)